exporter/cat1_view_helper.go

```diff
@@ -253,20 +262,14 @@ func hasPreferredCode(pc models.Concept) bool {
 	return pc.Code != "" && pc.CodeSystem != ""
 }
 
+func hasLaterality(l models.Laterality) bool {
```

Comment:
hasLaterality is more parallel to hasPreferredCode, which are helper functions used for the _code.xml helper function, working on a CodeDisplay object. It's not parallel to the hasSetOrdinality or hasSetSeverity methods, which were ported to the relevant HasEntry interface implementations.